definitions/contested/json/CaseEvent/CaseEvent.json

```diff
@@ -164,7 +164,7 @@
     "PreConditionState(s)": "schedulingAndHearing",
     "PostConditionState": "prepareForHearing",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/case-orchestration/documents/hearing",
-    "RetriesTimeoutURLAboutToSubmitEvent": "3,4,5",
+    "RetriesTimeoutURLAboutToSubmitEvent": "0",
```

Comment:
I believe this means it will timeout after 0 seconds, you can remove this altogether if you don't want any retries

Reply:
We also use this in PBA payments to force no retries. I believe this was the fix for duplicate payments. This is something thats I was experimenting as a fix (but encountered build issues) - and since then vivek has confirmed as an existing bug with the callback